internal/saucecloud/cloud.go

```diff
@@ -48,6 +49,7 @@ type CloudRunner struct {
 	RDCArtifactDownloader download.ArtifactDownloader
 
 	interrupted bool
+	DryRun      bool
```

Comment:
I'm still learning the code base so I might not fully understand all the project abstractions so this may be a silly question. Why move the DryRun attribute from `Project` to `CloudRunner`?

Reply:
I think currently it seems to be a good choice to have `DryRun` mode for each type of runner on sauce cloud. And I think it'd be better to move `DryRun` to `CloudRunner` 🤔